docs/user-guide/etendo-copilot/getting-started.md

```diff
@@ -0,0 +1,143 @@
+
+
+### Agent: Your Assistant
+
+The Agent serves as your virtual assistant, making on-the-fly decisions about which Tool is best suited to respond to a particular query. This intelligent decision-making ensures that you receive the most accurate and efficient assistance.
+
+### Tools: Your Specialized Partners
+
+Each Tool represents a separate independent project, designed to excel at specific tasks. Whether it is code translation, text analysis, or data manipulation, our collection of Tools work in harmony to deliver unparalleled support.
+
+###  Key Features:
+
+![Copilot Chat2](/assets/user-guide/etendo-copilot/getting-started/copilot-chat2.png){align=right  width="300" }
+
+
+- **Effortless Integration**: Etendo Copilot seamlessly integrates into your environment, adding an extra layer of intelligence to your workflow.
+
+- **On-Demand Assistance**: Send your queries to Etendo Copilot, and the Agent will guide you towards the most suitable Tool for the job.
+
+- **Diverse Expertise**: Our ever-growing selection of Tools covers a wide range of domains, ensuring you always have the right solution at your fingertips.
+
+- **Open AI Assistants**: Copilot is integrated with the Assistants technology developed by Open AI, allowing you to set up your assistants, trained with your own knowledge base, able to generate and interpret new code, and use the specific tools already distributed by Etendo or new ones.  
+
+### Copilot Chat
+
+In the Etendo Classic navigation bar, you'll find a Copilot icon that leads you to the chat pop-up.
+
+![Copilot Navbar](/assets/user-guide/etendo-copilot/getting-started/copilot-navbar.png)
+
+Here, you can select an Copilot App and engage in a conversation with it. Copilot facilitates communication with both `Langchain Agent` and `Open AI Assistant` types.
+
+### Copilot App Window
+
+The Copilot App window `Application`>`service`>`Copilot`>`Copilot App` allows you to define and configure applications:
+
+<figure markdown>
+  ![Copilot App](/assets/user-guide/etendo-copilot/getting-started/copilot-app.png)
+  <figcaption> Open AI assistant type example</figcaption>
+</figure>
+
+
+- *Name:* Copilot app name
+- *Description:* Copilot app description
+- *App Type:* Langchain Agent or Open AI Assistant
+
+*Open AI Assistant*
+    These applications harness OpenAI technology to offer assistance in a variety of tasks, from natural language processing to complex calculations.
+
+*Langchain Agent*
+    These applications can perform specific tasks in natural language and provide contextualized responses, allowing the implementation of multiple AI models, use of a proprietary vector database and internal memory management.
+
```

Comment:
Perhaps it should be explained that, currently, openai is for "asking things" and langchain is for executing the tools. In other words, clarify that if you use openai, the tools will not be executed (for the moment).